PowerShellForGitHub.psd1

```diff
@@ -204,7 +203,9 @@
     # RequiredAssemblies = @()
 
     # Script files (.ps1) that are run in the caller's environment prior to importing this module.
-    # ScriptsToProcess = @()
+    ScriptsToProcess = @(
+        'OnModuleLoad.ps1'

```

Comment:
Did you try this? It looks like this approach _doesn't_ work (as the comment points out) as it runs the script _before_ the module gets imported into the environment.  You should have seen a failure when you tried to import the module in a clean PowerShell session.

It looks like instead of doing it this way, you should just put `OnModuleLoad.ps1`  in the `NestedModules` list below `GitHubConfiguration.ps1`.